exercises/anagram/test/Tests.hs

```diff
@@ -1,40 +1,140 @@
+               , subject     = "corn"
+               , candidates  = ["corn", "dark", "Corn", "rank", "CORN", "cron", "park"]
+               , expected    = ["cron"]
+               }
+        , Case { description = "does not detect non-anagrams with identical checksum"
+               , subject     = "mass"
+               , candidates  = ["last"]
+               , expected    = []
+               }
+        , Case { description = "detects anagrams case-insensitively"
+               , subject     = "Orchestra"
+               , candidates  = ["cashregister", "Carthorse", "radishes"]
+               , expected    = ["Carthorse"]
+               }
+        , Case { description = "detects anagrams using case-insensitive subject"
+               , subject     = "Orchestra"
+               , candidates  = ["cashregister", "carthorse", "radishes"]
+               , expected    = ["carthorse"]
+               }
+        , Case { description = "detects anagrams using case-insensitve possible matches"
+               , subject     = "orchestra"
+               , candidates  = ["cashregister", "Carthorse", "radishes"]
+               , expected    = ["Carthorse"]
+               }
+        , Case { description = "does not detect a word as its own anagram"
+               , subject     = "banana"
+               , candidates  = ["Banana"]
+               , expected    = []
+               }
+        , Case { description = "does not detect a anagram if the original word is repeated"
+               , subject     = "go"
+               , candidates  = ["go Go GO"]
+               , expected    = []
+               }
+        , Case { description = "anagrams must use all letters exactly once"
+               , subject     = "tapper"
+               , candidates  = ["patter"]
+               , expected    = []
+               }
+        , Case { description = "eliminates anagrams with the same checksum"
+               , subject     = "mass"
+               , candidates  = ["last"]
+               , expected    = []
+               }
+        , Case { description = "detects unicode anagrams"
+               , subject     = "ΑΒΓ"
+               , candidates  = ["ΒΓΑ", "ΒΓΔ", "γβα"]
+               , expected    = ["ΒΓΑ", "γβα"]
+               }
+        , Case { description = "eliminates misleading unicode anagrams"
+               , subject     = "ΑΒΓ"
+               , candidates  = ["ABΓ"]
+               , expected    = []
+               }
+        , Case { description = "capital word is not own anagram"
+               , subject     = "BANANA"
+               , candidates  = ["Banana"]
+               , expected    = []
+               }
+        , Case { description = "anagrams must use all letters exactly once"
+               , subject     = "patter"
+               , candidates  = ["tapper"]
+               , expected    = []
+               }
+        , Case { description = "accepts string arguments"
```

Comment:
I see you got these from "string argument cases" section in x-common, I personally don't understand what they are for... maybe I need to ask. this particular case is a duplicate of an above case